src/i18n/i18n-store.ts

```diff
@@ -19,7 +19,7 @@ interface I18NState {
 }
 
 export const useI18nStore = create<I18NState>()((set, get) => ({
-	language: "de",
+	language: "en",
```

Comment:
is this on purpose?

Reply:
I just changed it for testing. Changed it back now